From: Carlos Garnacho <carlosg@gnome.org>
Date: Fri, 21 Oct 2022 17:23:36 +0200
Subject: core: Drop unnecessary code

This is already handled by the meta_compositor_grab_begin/end
calls, for the Wayland cases.

Forwarded: https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2683
Applied-upstream: 44.beta, commit:f2ff3172060feb9d41ce4cd2e95e5793c3d2daac
---
 src/core/events.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/core/events.c b/src/core/events.c
index aaf6f49..b3da428 100644
--- a/src/core/events.c
+++ b/src/core/events.c
@@ -244,15 +244,9 @@ meta_display_handle_event (MetaDisplay        *display,
     {
       MetaCompositor *compositor = meta_display_get_compositor (display);
 
-#ifdef HAVE_WAYLAND
-      if (wayland_compositor)
-        meta_display_sync_wayland_input_focus (display);
-#endif
-
       if (!display->grabbed_in_clutter && has_grab)
         {
           display->grabbed_in_clutter = TRUE;
-          meta_display_cancel_touch (display);
           meta_compositor_grab_begin (compositor);
         }
       else if (display->grabbed_in_clutter && !has_grab)
